couchbase/README.md

```diff
@@ -1,5 +1,5 @@
 # Couchbase Integration
-{{< img src="integrations/couchbase/couchbase_graph.png" alt="couchbase graph" responsive="true" popup="true">}}
```

Comment:
why did you remove this?

Reply:
Good question, it turns out that we need a determinable URL to display images in the new declarative integration tiles. Unfortunately, the URL for the images that are already on the app are linked to a version number we can't figure out. For example, the URL for the couchbase image: https://static.datadoghq.com/static/v/34.403017/images/setup/couchbase/snapshot.png

That specific image also happens to be in a private repo so we can't link directly to it.

We can move the image to a place that has a determinable URL, such as this repo. Maybe we can make another folder named `images/` and place the images in there. However, I am not sure placing sample images in this repo is a good idea.

Reply:
According to https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#add-images-and-logos screenshots are supposed to live in this repo, under `/images`

Reply:
I'll grab the graph we use 👍